src/main/java/repository/SurveyRepository.java

```diff
@@ -0,0 +1,13 @@
+package repository;
+
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import survey.Survey;
+
+import java.util.List;
+
+@RepositoryRestResource(collectionResourceRel = "survey", path = "survey")
+public interface SurveyRepository extends PagingAndSortingRepository<Survey, Long> {
+    List<Survey> findByName(@Param("name") String name);
```

Comment:
Should we have a find by ID as well?

Reply:
I don't think it is necessary. The user will always have a name of the survey.